src/components/playback/playbackmanager.js

```diff
@@ -2443,7 +2452,7 @@ class PlaybackManager {
         }
 
         function playAfterBitrateDetect(maxBitrate, item, playOptions, onPlaybackStartedFn, prevSource) {
-            const startPosition = playOptions.startPositionTicks;
+            const startPosition = item.UserData.PlaybackPositionTicks && playOptions.isFirstItem ? item.UserData.PlaybackPositionTicks : playOptions.startPositionTicks;
```

Comment:
Doesn't this prevent us from restarting playback?

I haven't thought much about this, but we could use `playOptions.startPositionTicks` to control resume/restart/seek.
If `playOptions.startPositionTicks` isn't set, we use `item.UserData.PlaybackPositionTicks` - resuming.
Otherwise we use `playOptions.startPositionTicks` - restarting or using a time link.

Reply:
Yea it does seem that way, I'm mostly lacking information on where to pipe the information to reason what to do.